core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -111,17 +114,45 @@ private Path getFilePath(PageId pageId) {
         Long.toString(pageId.getPageIndex()));
   }
 
-  @Override
-  public void close() {
+  /**
+   * @param path path of a file
+   * @return the corresponding page id, or null if the file name does not match the pattern
+   */
+  private PageId getPageId(Path path) {
+    Path parent = Preconditions.checkNotNull(path.getParent());
+    if (!Paths.get(mRoot).equals(parent.getParent())) {
+      return null;
```

Comment:
Should we warn on files we were unable to recover? Also should we delete them?

Reply:
Added some warnings.